player/lua/osc.lua

```diff
@@ -2058,7 +2058,8 @@ function process_event(source, what)
 
             if n == 0 then
                 --click on background (does not work)
-            elseif n > 0 and not (elements[n].eventresponder[source .. "_" .. what] == nil) then
+            elseif n > 0 and not (n > #elements) and
+                not (elements[n].eventresponder[source .. "_" .. what] == nil) then
```

Comment:
In case it wasn't clear, Lua has a `~=` inequality operator. Also, just evaluating a value as bool is the same as checking for inequality to both `nil` and `false`.